packages/@lwc/template-compiler/src/parser/index.ts

```diff
@@ -282,22 +266,19 @@ export default function parse(source: string, state: State): TemplateParseResult
         const validRoots = documentFragment.childNodes.filter(
             (child) =>
                 treeAdapter.isElementNode(child) ||
-                (treeAdapter.isTextNode(child) &&
-                    treeAdapter.getTextNodeContent(child).trim().length)
+                (treeAdapter.isTextNode(child) && child.value.trim().length)
```

Comment:
What is the difference between `child.value` and `treeAdapter.getTextNodeContent(child)`? Why the change?